unbreak i386 build;  fixed upstream as part of Apple related and coverity fixes
https://github.com/web-eid/libpcsc-cpp/pull/25

Index: lib/libelectronic-id/lib/libpcsc-cpp/src/SCardCall.hpp
--- lib/libelectronic-id/lib/libpcsc-cpp/src/SCardCall.hpp.orig
+++ lib/libelectronic-id/lib/libpcsc-cpp/src/SCardCall.hpp
@@ -47,7 +47,7 @@ void SCardCall(const char* callerFunctionName, const c
                const char* scardFunctionName, Func scardFunction, Args... args)
 {
     // TODO: Add logging - or is exception error message enough?
-    const uint32_t result = scardFunction(args...);
+    const LONG result = scardFunction(args...);
 
     // TODO: Add more cases when needed.
     switch (result) {
